docs/sources/articles/networking.md

```diff
@@ -499,7 +505,8 @@ want to configure `eth0` via Router Advertisements you should set:
 ![](/article-img/ipv6_basic_host_config.svg)
 
 Every new container will get an IPv6 address from the defined subnet. Further
-a default route will be added via the gateway `fe80::1` on `eth0`:
+a default route will be added on `eth0` via the address specified by the daemon
```

Comment:
you mention `eth0` twice in this sentence, but if i grok it right, the first time you're talking about the `eth0` in the container, and the second one refers to the default network on the host.

on quite a few of my systems, I don't have an `eth0` - RHEL seems to use some obfuscated device name.

So I would suggest making the first reference be explicit: `eth0 in the container` and the second could be changed to the `host's default gateway`